Library/Homebrew/cask/audit.rb

```diff
@@ -270,6 +271,16 @@ def bad_osdn_url?
       bad_url_format?(/osd/, [%r{\Ahttps?://([^/]+.)?dl\.osdn\.jp/}])
     end
 
+    def check_url_verification
+      return unless cask.url.verified
+
+      url = cask.url.to_s.sub(%r{^(http|https)://}, "")
+      verification = cask.url.verified.sub(%r{^(http|https)://}, "")
+      return if url.start_with?(verification)
+
+      add_error "URL verification failed. See https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-verified"
```

Comment:
`URL verification failed` is too generic. It should roughly explain what that means.